### ISSUE TITLE
Loading entries does not work with Github backend (master branch regression)

### ISSUE THREAD

**Issue by melbourne2991** [netlify/netlify-cms#2629]
Related discussion here: 
https://github.com/netlify/netlify-cms/pull/2523#issuecomment-526965166

On the master branch I am getting a 401 when loading entries because the incorrect URL is being requested:

```
/.netlify/git/github/repos//contents/src/pages/blog?ts=1567345646611&ref=master
```

Should be:

```
/.netlify/git/github/contents/src/pages/blog?ts=1567345646611&ref=master
```

Probably related the open authoring changes as there seems to have been a few related changes here: 
https://github.com/netlify/netlify-cms/pull/2523

To reproduce:
1. Deploy a fresh netlifyCMS backed by Github
2. Enable Identity / Gitgateway
3. Navigate to `/admin`
4. You should see an error

**Comment by erquhart** [comment]
@melbourne2991 this may have been addressed in #2523, does it seem to be working for you in the latest beta?

**Comment by erquhart** [comment]
Oh you posted this _after_ that fix, nvm - looking at your PR now 👍
